.circleci/config.yml

```diff
@@ -9,23 +9,22 @@ jobs:
           keys:
             - node-cache-{{ checksum "package-lock.json" }}
             - node-cache
-      - run: npm install
-      - run: npx lerna bootstrap
-      - run: npm run build
+      - run: yarn install
+      - run: yarn build
       - run:
           name: prettier
-          command: npm run prettier:check
+          command: yarn prettier:check
       - run:
           name: lint
-          command: npm run eslint -- --format junit -o reports/junit/js-lint-results.xml
+          command: yarn eslint -- --format junit -o reports/junit/js-lint-results.xml
       - run:
           name: tests
-          command: npm run test -- --no-cache --ci --runInBand --reporters=default --reporters=jest-junit
+          command: yarn test -- --no-cache --ci --runInBand --reporters=default --reporters=jest-junit
           environment:
             JEST_JUNIT_OUTPUT: 'reports/junit/js-test-results.xml'
       - run:
           name: types
-          command: npm run ts:check
+          command: yarn ts:check
       - save_cache:
```

Comment:
you may need to update this with the same key as above, also I'm not sure this may work as expected, please check circleci is actually using the cache, before I have to manually set the yarn cache folder (`yarn install --cache-folder ~/.cache/yarn`) and save that folder here but I haven't use yarn in a while

Reply:
Def need to update to use the same key.